database/pgsql/migrations/00020_languagelayer_lineage_pk.go

```diff
@@ -0,0 +1,14 @@
+package migrations
+
+import "github.com/remind101/migrate"
+
+func init() {
+	RegisterMigration(migrate.Migration{
+		ID: 20,
+		Up: migrate.Queries([]string{
+			`ALTER TABLE LanguageLayer DROP CONSTRAINT IF EXISTS languagelayer_pkey`,
```

Comment:
is this a generic, default constraint? I don't see it in my deployment:

```
postgres=# \d languagelayer
                         Table "public.languagelayer"
       Column       |          Type          | Collation | Nullable | Default 
--------------------+------------------------+-----------+----------+---------
 layer_id           | integer                |           | not null | 
 layer_name         | character varying(128) |           | not null | 
 component_data     | bytea                  |           | not null | 
 removed_components | text[]                 |           |          | 
 lineage            | character varying      |           |          | 
Indexes:
    "languagelayer_layer_name_key" UNIQUE CONSTRAINT, btree (layer_name)
```

Reply:
`languagelayer_pkey` is the name of the constraint after `ALTER TABLE LanguageLayer ADD PRIMARY KEY (layer_name, lineage)` is applied, the default name of a primary key looks to be `<table name>_pkey`

Possibly added to the migration so that it would not cause errors if re-ran.